src/console-errors/wiiu/support-codes/codes/160-sys.js

```diff
@@ -3,9 +3,9 @@ const SupportCode = require('../../../common/support-code');
 module.exports = {
 	'0101': new SupportCode('Missing name', 'Missing description', 'Missing message', 'Missing fix', 'Missing link'),
 	'0102': new SupportCode('Missing name', 'Missing description', 'Missing message', 'Missing fix', 'Missing link'),
-	'0103': new SupportCode('Missing name', 'Missing description', 'Missing message', 'Missing fix', 'Missing link'),
+	'0103': new SupportCode('Missing name', 'MLC corruption error', 'There is a problem with the system memory.\n\nFor help, make a note of the error code and visit\nsupport.nintendo.com', 'Usually happens on 32GB Wii U models having a Hynix chip,\nthis chip has a problem that slowly corrupts its data overtime,\nthere is nothing much to do except backing up the MLC\nand replacing the Hynix chip entierly.', 'Missing link'),
```

Comment:
Since not every user will know which chip they have, making the link field point to https://github.com/GaryOderNichts/WiiUIdent along with detailing in the error description that this tool can help identify which chip you're using would be very helpful

Reply:
I'll do that, thanks for the review :+1: (sorry if I didn't viewed your review yesterday :'))

Reply:
Done